src/app/story/directives/hero.component.ts

```diff
@@ -46,6 +47,12 @@ import { StoryModel } from '../../shared';
              [working]="story.isPublishing" (click)="togglePublish()" orange=1>
             {{story.publishedAt ? 'Unpublish' : 'Publish'}}
           </publish-button>
+          <p *ngIf="!story.changed() && !story.publishedAt">
+            Will be queued up for publishing {{story.releasedAt ? ' at ' + formatDate(story.releasedAt) : 'immediately'}}
```

Comment:
I like @cavis's ideas if we want to change the button text -- otherwise, if we want to stick with hint text below the button, I'd vote for brevity over explanation of process-- aka, just say "Will be published on {{story.releasedAt}}" (skip the mention of queuing, and also skip the time). One thing with the text as is is that if `releasedAt` is deliberately set in the past, the text sounds a little funny since it is future-focused. Not sure if this way would be any better.